src/client/src/App/Credit/common/RfqTicketHeader.ts

```diff
@@ -15,9 +15,7 @@ export const DirectionContainer = styled.div<WithDirection & WithTerminated>`
   flex: 0 0 48px;
   background-color: ${({ theme, direction, terminated }) =>
     terminated
-      ? theme.name === ThemeName.Dark
-        ? "#222730"
-        : theme.primary[2]
+      ? theme.primary[2]
```

Comment:
I take it this all ratified with UX .. looks like a funky non-standard bg before, anyway ...

Reply:
yeah, we've had background talks 🔲